Jantimizer/QueryGenerator/QueryGenerators/PostgresGenerator.cs

```diff
@@ -6,7 +6,7 @@
 using System.Text;
 using System.Threading.Tasks;
 
-namespace QueryOptimiser.QueryGenerators
```

Comment:
Would it not make more sense to either have it part of the query optimiser namespace or as a seperate project. It feels kinda off to have it's own namespace while it's a part of a bigger project.